dataland-frontend/tests/e2e/support/TestUtility.ts

```diff
@@ -2,6 +2,7 @@ import { Suite } from "mocha";
 
 export interface ExecutionConfig {
   executionEnvironments: Array<ExecutionEnvironment>;
+  onlyExecuteOnReset: boolean;
```

Comment:
Some way to have a default value of false here?

Reply:
We could make this parameter optional I guess. However, this is intentional. My reasoning is that when using describeIf the user should be aware of all the options.